cmd/registry/patch/apply.go

```diff
@@ -223,6 +225,8 @@ func (task *applyBytesTask) Run(ctx context.Context) error {
 
 	log.FromContext(ctx).Infof("Applying %s", task.resource())
 	switch header.Kind {
+	case "Project":
+		return applyProjectPatchBytes(ctx, task.adminClient, task.bytes)
```

Comment:
It looks to me like everything in this PR is a setup for this handling this new case. That LGTM, and I guess if we are running with a hosted service, the request will just fail if a `Project` YAML is included. Also LGTM (thinking out loud).